buildSrc/settings.gradle.kts

```diff
@@ -1,7 +1,11 @@
 dependencyResolutionManagement {
+    repositories {
+        google()
+        mavenCentral()
+    }
     versionCatalogs {
-        create("appLibs") {
-            from(files("../gradle/dependencies.toml"))
+        create("libs") {
+            from(files("../gradle/libs.versions.toml"))
```

Comment:
Moved to a single `.toml` file. `libs.version.toml` has support in the IDE and is displayed in the file tree when using the `Android` project view.

Reply:
Any idea why this doesn't work for submodules? i.e `foundation` module has `ui`, `async` submodules, but theit dependencies don't show up

Reply:
@catadarjan-tapptitude Not sure what do you mean when you say their dependencies don't show up 🤔 Could you please expand on this?